dex-it-common/src/main/scala/com/wavesplatform/dex/it/api/websockets/HasWebSockets.scala

```diff
@@ -8,16 +8,29 @@ import akka.actor.ActorSystem
 import akka.http.scaladsl.model.ws.Message
 import akka.stream.Materializer
 import com.google.common.primitives.Longs
+import com.softwaremill.diffx.{Derived, Diff}
+import com.wavesplatform.dex.api.websockets.{WsAddressState, WsOrderBook}
 import com.wavesplatform.dex.domain.account.KeyPair
+import com.wavesplatform.dex.domain.asset.AssetPair
 import com.wavesplatform.dex.it.docker.DexContainer
+import com.wavesplatform.dex.test.matchers.DiffMatcherWithImplicits.getDiff
 import mouse.any._
 import org.scalatest.{BeforeAndAfterAll, Suite}
+import play.api.libs.json.Json
 
 trait HasWebSockets extends BeforeAndAfterAll { _: Suite =>
 
   implicit protected val system: ActorSystem        = ActorSystem()
   implicit protected val materializer: Materializer = Materializer.matFromSystem(system)
 
+  implicit protected val derivedAddressStateDiff: Diff[WsAddressState] = Derived {
+    getDiff[WsAddressState]((l, r) => l.balances == r.balances && l.orders == r.orders)
```

Comment:
Please use [ignore](https://github.com/softwaremill/diffx/blob/master/README.md#customization) and we will never forget to add a new field to comparison.

Reply:
Done